.buildkite/pipeline.yml

```diff
@@ -121,7 +121,6 @@ steps:
   - label: "Metabase Test Suite"
     key: "metabase-test-suite"
     depends_on: "build-chromium-linux-x86_64"
-    if: build.branch == "master"
```

Comment:
did this just to get the buildkite step to run for this PR, but is there a reason we shouldn't be running the metabase tests always?

Reply:
(current answer: they take a while to run.)